tests/tracking/test_model_registry.py

```diff
@@ -2,97 +2,51 @@
 Integration test which starts a local Tracking Server on an ephemeral port,
```

Comment:
Refactored this file to test `MlflowClient` with both `FileStore` and `SqlAlchemyStore` and ensure their behaviors match.